apps/public-reference/pages/applications/preferences/general.tsx

```diff
@@ -9,28 +9,30 @@ import FormsLayout from "../../../layouts/forms"
 import { useForm } from "react-hook-form"
 import { AppSubmissionContext } from "../../../lib/AppSubmissionContext"
 import ApplicationConductor from "../../../lib/ApplicationConductor"
-import { useContext, useMemo } from "react"
+import { useContext, useMemo, useEffect } from "react"
 
 export default () => {
   const { conductor, application, listing } = useContext(AppSubmissionContext)
   const currentPageStep = 4
 
+  useEffect(() => {
+    if (!application.preferences.none) {
+      // Only describe the General Pool if no preferences were selected
+      Router.push("/applications/review/demographics")
+    }
+  }, [application])
```

Comment:
This dependency should _probably_ be `application.preferences` or `application.preferences.none` - we don't want this effect to run only when `application` changes?

Reply:
I'm not sure TBH…ideally this URL never comes up in the flow if it's configured right assuming a preference was selected, so this is simply an escape hatch.